dateutil/_common.py

```diff
@@ -24,11 +24,20 @@ def __eq__(self, other):
             return False
         return True
 
-    __hash__ = None
+    def __hash__(self):
+        return hash((
+          self.weekday,
+          self.n,
+        ))
+
+    def __ne__(self, other):
+        return not (self == other)
 
     def __repr__(self):
         s = ("MO", "TU", "WE", "TH", "FR", "SA", "SU")[self.weekday]
         if not self.n:
             return s
         else:
             return "%s(%+d)" % (s, self.n)
+
+# vim:ts=4:sw=4:et
```

Comment:
In the future I think maybe don't add this comment at the bottom. It's already there in a few other files, so it's not a big deal, but I think editor-specific detritus probably doesn't belong in upstream repos.